test/Microsoft.Sbom.Api.Tests/Config/ConfigSanitizerTests.cs

```diff
@@ -281,4 +286,35 @@ public void ShouldNotOverridePackageSupplierIfProvided_Succeeds()
 
         Assert.AreEqual(actualOrg, config.PackageSupplier.Value);
     }
+
+    [TestMethod]
+    [DataRow(ManifestToolActions.Validate)]
+    [DataRow(ManifestToolActions.Generate)]
+    public void ConfigSantizer_Validate_ReplacesBackslashes_Linux(ManifestToolActions action)
+    {
+        if (!isWindows)
```

Comment:
My original suggestion was going to be that we should use something like `[IgnoreIf(!isWindows)]` but as far as I can tell that doesn't exist in MSTest.

Instead I'll suggest that we test replacing forward slashes when testing on windows (I know the tests only run on linux ATM, but it would run locally and be good future-proofing).

Reply:
So at the moment there is no logic in the PathUtils.cs that would replace forward slashes to backslashes on windows. I left this out because both are supported on windows so it wouldn't cause an issue if either are provided.

Reply:
It would be visible in telemetry which the user could see if they output telemetry to a file path or use -V verbose. It is a simple change. We would just use path.Value = path.Value.Replace('\\', Path.AltDirectorySeparatorChar). The AltDirectorySeparatorChar is a forward slash on both linux and windows. The only concern would be that any tests that rely on Path.Join() when asserting values would fail since that would return values on backslashes. I ran a quick test and doing this would only require changes in 2 tests.

Reply:
My 2 cents: Since windows already supports / we should use that and make our lives simpler.

We should also use path.join whenever we join paths since that will be more correct than just string contactenation. If this causes problems in Windows, we should have our own PathJoin method in IFileSystemUtils, which does path.join internally and normalizes the path to / before returning.